bundle/camunda-saas-bundle/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM eclipse-temurin:17.0.8.1_1-jre
+FROM eclipse-temurin:21.0.1_12-jre
```

Comment:
My assumption that similar to [these steps to reproduce](https://github.com/camunda/connectors/issues/1556#issuecomment-1850539190) could be sufficient.

Reply:
AFAIK the `@Scheduled` annotation kicked in @chillleader when you started in locally right? I think the virtual threads topic is separate from that.

Reply:
Yes, my understanding is that virtual threads are bound to Spring Boot 3.2 and we're currently still on 3.1. I checked it in SaaS and scheduling is working as expected

Reply:
It's more of a point that virtual threads is a core feature in Java 21, meaning there's high probability that users will enable it by default. That's where we need to make sure that it won't brake since there are evidence that `@Scheduled` and virtual threads are for some reasons not dealing together very well.

Reply:
@igpetrov Seems like it was an issue inside spring: https://github.com/spring-projects/spring-boot/issues/38846

Reply:
But the user reported that issue while using Spring Boot 3.2 right? In 3.1 virtual threads are not supported. I don't see how we can test this scenario without migrating to 3.2, and this migration is currently [failing](https://github.com/camunda/connectors/pull/1514), so I believe this is out of scope

Reply:
@sbuettner, good catch! The GH issue is even with my screenshots 😆 - outcome of the earlier discussion at https://github.com/camunda/connectors/issues/1556.